tick/array/tests/serializer.py

```diff
@@ -0,0 +1,35 @@
+# License: BSD 3 clause
+
+import unittest
+
+import tick.array
+
+from tick.dataset import fetch_tick_dataset
+
+from tick.array.build.array import tick_double_array_to_file, tick_double_sparse2d_to_file
+from tick.array.build.array import tick_double_array_from_file, tick_double_sparse2d_from_file
+
+from tick.optim.solver.tests.solver import TestSolver
+
+class Test(TestSolver):
+  def test_set_model(self):
+
+    features = 'covtype.features.cereal'
+    lables   = 'covtype.labels.cereal'
+    dataset  = 'binary/covtype/covtype.trn.bz2'
+
+    X, y = fetch_tick_dataset(dataset)
+    print(type(X))
+    print(X.toarray())
+    
+    tick_double_array_to_file(lables, y)
+    tick_double_sparse2d_to_file(features, X)
+    
+    y1 = tick_double_array_from_file(lables)
+    X1 = tick_double_sparse2d_from_file(features)
+    
+    print("X1 type: " + str(type(X1)))
+    print(X1.toarray())
+
+if __name__ == '__main__':
+  unittest.main()
```

Comment:
You can use `np.test.assert_array_equal` to check array equality

Reply:
That function is never called, it was for debugging GC while trying to figure out if SparseArray2d was being deleted properly. Can remove though,

Reply:
k